saleor/core/management/commands/populatedb.py

```diff
@@ -60,3 +70,5 @@ def handle(self, *args, **options):
             else:
                 self.stdout.write(
                     'Superuser already exists - %(email)s' % credentials)
+        if not options['withoutsearch']:
+            self.populate_search_index()
```

Comment:
Not sure we should call one management from another. I think I'd rather document how various parts play together.